Remotion/Web/Development.WebTesting.IntegrationTests/BrowserSessionTest.cs

```diff
@@ -75,12 +75,13 @@ public void GeckoDriver_DoesNotSupportBrowserLogs ()
 
       var home = Start();
 
-      var selenium = (IWebDriver)home.Context.Browser.Driver.Native;
+      var js = JavaScriptExecutor.GetJavaScriptExecutor(home.Context.Browser);
```

Comment:
I see you just took the variable name from the Test-method further down. No need to change the name now sinse it's the same as below, but just for learning: non-abbreviated names are preferred when writing new code.